### ISSUE TITLE
readme

### ISSUE THREAD

**Issue by chengzhangsuper** [ReinforceLove/MediDRNet#1]
Could you please tell me how to run your code? What is the format of the list files for the training set and the test set?
